TestResultSummaryService/routes/index.js

```diff
@@ -30,6 +30,7 @@ app.get( '/getTestPerPlatform', wrap( require( "./getTestPerPlatform" ) ) );
 app.get( '/getTopLevelBuildNames', wrap( require( "./getTopLevelBuildNames" ) ) );
 app.get( '/getTotals', wrap( require( "./getTotals" ) ) );
 app.get( '/populateDB', wrap( require( "./populateDB" ) ) );
+app.get( '/getDeepSmithDataCompare', wrap( require( "./getDeepSmithDataCompare" ) ) );

```

Comment:
Please name this API more generic as it can be used for other test output compare. For example, compareTestOutput